Tests/test_BWA_tool.py

```diff
@@ -91,27 +91,33 @@ def test_index(self):
         cmdline = BwaIndexCommandline(bwa_exe)
         cmdline.set_parameter("infile", self.reference_file)
         cmdline.set_parameter("algorithm", "bwtsw")
+        try:
+            stdout, stderr = cmdline()
+        except ApplicationError as err:
+            self.assertTrue(err.returncode == 0,
```

Comment:
Style wise, how about ``self.assertEqual(err.returncode, 0, ...)`` here?

However, including this assert puzzles me. If we get an ``ApplicationError``, surely we expect the return code to be non-zero?